crowd_anki/export/anki_exporter.py

```diff
@@ -28,7 +28,7 @@ def export_to_directory(self, deck: AnkiDeck, output_dir=Path("."), copy_media=T
         deck = deck_initializer.from_collection(self.collection, deck.name)
         self.last_exported_count = deck.get_note_count()
 
-        deck_filename = deck_directory.joinpath(deck_fsname).with_suffix(DECK_FILE_EXTENSION)
+        deck_filename = deck_directory.joinpath('deck').with_suffix(DECK_FILE_EXTENSION)
```

Comment:
Please extract the actual name used to a constant in https://github.com/Stvad/CrowdAnki/blob/master/crowd_anki/utils/constants.py

Reply:
good point, done in the latest commit.